### ISSUE TITLE
Metamask (extension) connection to Trezor wallet

### ISSUE THREAD

**Issue by Wmartin417** [arkenfox/user.js#1352]
I apologize for maybe not providing enough info as this is the first issue I have every raised in a github repo before, going to go off other issues I see and provide similar info. I understand that most people may not have a hardware wallet (for crypto) and might be somewhat difficult to troubleshoot as a result. 

- browser version
96.0.1

- Steps to Reproduce
Install trezor suite and trezor bridge, plug in trezor, run trezor suite
In firefox, install the Metamask extension, sign in to wallet, click connect hardware wallet, select trezor.

- actual result
Metamask does not seem to see the bridge running on the host PC. Error: Transport is missing.

- expected result
Metamask connects to trezor bridge running on host and allows for connection between the wallet plugged in the usb port

- anything else you deem worth mentioning
Connecting to my trezor via trezor's webUI works and seems to pick up on the bridge running, but not metamask. Maybe there is a setting that is disrupting extension's ability to see a program running on the host?

**Comment by Thorin-Oakenpants** [comment]
links
- https://addons.mozilla.org/en-US/firefox/addon/ether-metamask/
- https://wiki.trezor.io/Trezor_Bridge

Sorry for the delay, and it might be some time before I get to this (and not sure how I can test)

The first thing is to make sure it works in Firefox 96 - in a new profile without any other extensions or pref changes
Then confirm that it breaks when you add the arkenfox user.js
This means its not an upstream issue and IS caused by a pref in arkenfox

I'm not sure what the cause could be offhand
- RFP (4501)
- referers (1601)
- UNC paths (0703)
- extension scope (2660)
- security.OCSP.require (1212) <-- this one is known to break some VPN extensions

otherwise, the quickest way is to use a new profile and use a binary method to narrow it down - that is
- new profile, add half of arkenfox, start FF and test - now you know which half the problem is in
- if it worked, then add the next quarter, restart FF and test - now you know which quarter

https://github.com/arkenfox/user.js/wiki/1.4-Troubleshooting#small_orange_diamond-manual-troubleshooting <-- the old link while it lasts

**Comment by Thorin-Oakenpants** [comment]
IANAE - but the bridge is running on localhost, right?

**Comment by Wmartin417** [comment]
> The first thing is to make sure it works in Firefox 96 - in a new profile without any other extensions or pref changes
Then confirm that it breaks when you add the arkenfox user.js
This means its not an upstream issue and IS caused by a pref in arkenfox

Well, thought I had tested this, but apparently not well enough. Before I just opened a incognito window with all but metamask disabled, the problem still persisted. 

Disabling my extensions in Firefox settings instead of relying on an incognito window allowed me to narrow down that it was actually ublock causing the issue. 

> The first thing is to make sure it works in Firefox 96 - in a new profile without any other extensions or pref changes
Then confirm that it breaks when you add the arkenfox user.js
This means its not an upstream issue and IS caused by a pref in arkenfox

Apologies, I should have looked at the troubleshooting section, could have just have read that and not bothered anyone. I was stuck in 1.3 and override recipes and ctrl-f through arkenfox user.js. I'll take note to read the documentation a bit more in the future.

> IANAE - but the bridge is running on localhost, right?
Yes! It is. It is required to work with firefox as (allegedly, IANAE as well) does not support WebUSB. However, it is encourages as it is more of a secure option according to Trezor.

Thank you for your kind reply and again, apologies for wasting your time!
